test/models/report_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+class ReportTest < ActionController::TestCase
+  test "valid? returns false when report is too short" do
+    msg = "Report is too short (minimum is 100 characters)"
+    report = Report.new
+
+    assert_not report.valid?
+    assert_equal msg, report.errors.full_messages.first
+  end
+
+  test "valid? returns false when report is too big" do
+    msg = "Report is too long (maximum is 20000 characters)"
```

Comment:
if we move the `20_000` as a constant we can make this tests easier to maintain using that constant here instead of hardcoding the number.